website/templates/emails/reviews_submission_confirmation.txt.mako

```diff
@@ -0,0 +1,31 @@
+Hello ${user.fullname},
```

Comment:
Product team has specified all emails to be in html email template instead of txt. There is a ticket to change all current existing txt email to html, which is https://openscience.atlassian.net/browse/OSF-8770. This is fine for now. But we should stop writing any more txt email template.